### PR TITLE
[TEVA-1188] Anchor page to vacancies on form submit

### PR DESCRIPTION
**Pull request by cpjmcquillan**
This PR anchors both the job results page and the dashboard to the job results.

## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1188

## Changes in this PR:
- Form submit url is anchored to `#vacancies-hits` elements

### CODE REVIEW

**Review by cesidio — APPROVED** [review]

File: app/views/hiring_staff/organisations/_vacancies_awaiting_feedback.html.haml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-%section.govuk-tabs__panel
+%section.govuk-tabs__panel#vacancies-hits
```
Comment (alexbowen) [inline app/views/hiring_staff/organisations/_vacancies_awaiting_feedback.html.haml:1]: wouldn't it be better to put this line outside of all these templates in the presumably one parent template so all the duplication here is removed?

**Comment by alexbowen** [comment]
What is done in this PR means that a user using keyboard navigation cannot tab to the search inputs once they have used a filter

**Comment by alexbowen** [comment]
Or to rephrase this pr isn't incorrect but need in addition some clever use of aria tags and possibly extra link somewhere. Let's discuss tomorrow

File: app/views/shared/_skip_links.html.haml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+%a.govuk-skip-link{ href: "#main-content" } Skip to main content
+- if params[:jobs_search_form]
+  %a.govuk-skip-link{ href: "#vacancies-hits" } Skip to search results
```
Comment (cpjmcquillan) [inline app/views/shared/_skip_links.html.haml:3]: New line

**Review by alexbowen — APPROVED** [review]

**Pull request merged**